contracts/access_control/lib.rs

```diff
@@ -25,10 +31,20 @@ mod access_control {
     pub const HAS_ROLE_SELECTOR: [u8; 4] = [0, 0, 0, 3];
     pub const CHECK_ROLE_SELECTOR: [u8; 4] = [0, 0, 0, 5];
 
-    #[ink(storage)]
-    pub struct AccessControl {
+    pub const STORAGE_KEY: u32 = openbrush::storage_unique_key!(Data);
```

Comment:
I'm personally not a huge fan of defining storage keys this way, as it's less transparent what's going on and how to migrate (you need to change `Data` struct name, I would just use constant keys). But it's kinda the default way, so I leave it to your judgement which way to choose.